app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -9,29 +9,32 @@ package mozilla.lockbox.action
 import android.support.annotation.StringRes
 import mozilla.lockbox.flux.Action
 
-sealed class RouteAction : Action {
-    object Welcome : RouteAction()
-    object ItemList : RouteAction()
-    object Login : RouteAction()
-    object SettingList : RouteAction()
-    object AccountSetting : RouteAction()
-    object AutoLockSetting : RouteAction()
-    object Back : RouteAction()
-    object LockScreen : RouteAction()
-    object Filter : RouteAction()
-    data class ItemDetail(val id: String) : RouteAction()
-    data class OpenWebsite(val url: String) : RouteAction()
-    data class SystemSetting(val setting: SettingIntent) : RouteAction()
+sealed class RouteAction(
```

Comment:
_Discussed with @irrationalagent IRL ..._

Technically, it doesn't really matter if all `RouteAction`s are also `TelemetryAction`s.  It matters more for data science, and the resident data scientist doesn't see harm in this.

I'm inclined to leave this as changed here; @sashei @jhugman @eliserichards @ioana-farcas thoughts?